cli/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "4.7.0"
 authors = ["Phylum, Inc. <engineering@phylum.io>"]
 license = "GPL-3.0-or-later"
 edition = "2021"
-rust-version = "1.65.0"
+rust-version = "1.68.0"
```

Comment:
Very high MSRV, due to requirement of the `pin!` macro. But I do believe it should be fine for CLI since we build it ourselves anyway and very few people need to build from source.